static/js/uStates.js

```diff
@@ -17,51 +17,73 @@
   var path = d3.geoPath()
     .projection(projection);
 
-  var genres = ['pop', 'indie', 'country', 'classical', 'hiphop', 'metal']
+  var genres = ['pop', 'rock_metal', 'indie', 'hiphop', 'rnb', 'classical_jazz', 'electronic_dance', 'country_folk'];
+
   var colors = {
```

Comment:
[SC]
We are gonna use these colours for states, counties, venues, the bar chart, etc. So we don't have to create a new dict every time we want to use these colours, we can move these "constants" to a separate file and include it at the top of the file. 

Again, don't worry too much about it and I can do it for you tonight and show you what it would look like!